.github/ISSUE_TEMPLATE/blog_submission.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Blog submission request
+about: Submit one blog post to be published in our website
+title: 'blog: '
+labels: 'blog'
+assignees: ''
+
+---
+
+### Blog Metadata
+
+Please provide the following information:
+
+* Title:
+* Author:
```

Comment:
I think it would be helpful to include position / company as well, what do you think?